doc/tools/docgram/common.edit_mlg

```diff
@@ -1225,11 +1225,8 @@ show_zify: [
 ]
 
 scheme_kind: [
-| DELETE "Induction" "for" smart_global "Sort" sort_family
-| DELETE "Minimality" "for" smart_global "Sort" sort_family
-| DELETE "Elimination" "for" smart_global "Sort" sort_family
-| DELETE "Case" "for" smart_global "Sort" sort_family
-| [ "Induction" | "Minimality" | "Elimination" | "Case" ] "for" smart_global "Sort" sort_family
+| DELETE scheme_type "for" smart_global "Sort" sort_family
+| scheme_type "for" smart_global "Sort" sort_family
```

Comment:
Can drop these, together they are a no-op.

Reply:
Should I get rid of the scheme_kind all together?

Reply:
Or just get rid of DELETE?

Reply:
The second line is already present in `fullGrammar`, so no need to add it again here.  doc_gram_rsts should give an error if you add a production that's already in the grammar (if it's not, let me know).  Both lines can go.

Reply:
Yes, you can drop `scheme_kind` here.

Reply:
It seems doc_gram_rsts doesn't mention it.

Reply:
No bug--the DELETE on the previous line removed the production, so no error--my comment was mistaken.